build.sbt

```diff
@@ -215,9 +215,9 @@ val compileAndTest = "compile->compile;test->test"
 def dependenciesFor(version: String)(deps: (Option[(Long, Long)] => ModuleID)*): Seq[ModuleID] =
   deps.map(_.apply(CrossVersion.partialVersion(version)))
 
-lazy val coreProjectAggregates: Seq[ProjectReference] = if (sys.env.isDefinedAt("STTP_NATIVE")) {
+lazy val projectAggregatesWithNative: Seq[ProjectReference] = if (sys.env.isDefinedAt("STTP_NATIVE")) {
```

Comment:
this contains the native projects optionally, so maybe `projectWithOptionalNative` or sth like that?

Reply:
I renamed to `projectsWithOptionalNative` since they are more than one :)